packages/@extension-kid/toast-notifications/ToastNotification.ts

```diff
@@ -21,7 +21,7 @@ type ToastCallback = (
 interface ToastNotificationOptions {
   appearance?: ToastAppearance;
   autodismiss?: boolean;
-  description?: string;
+  description?: string | { id: string; values: Record<string, string> };
```

Comment:
Does it make sense to create an interface for this type of object?

Reply:
yes, 👍